Gems/Atom/Feature/Common/Assets/Materials/Types/DepthPass_WithPS.azsl

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+struct VSInput
+{
```

Comment:
Are the following structures shared / will be shared between several shaders so that can be moved to their own azsli file as done n other case for these structures?

Reply:
Quite possibly, but for now, the main focus of this refactor was to consolidate the SRG referencing code (not the interpolants). More cleanup will occur in future passes but I restricted this change to only what's on the path for future material canvas work